docs/data/joy/components/menu/MenuIconSideNavExample.js

```diff
@@ -78,12 +78,20 @@ function NavMenuButton({
           isOnButton.current = false;
         }}
         onKeyDown={handleButtonKeyDown}
-        sx={{
-          bgcolor: open ? 'neutral.plainHoverBg' : undefined,
-          '&:focus-visible': {
-            bgcolor: 'neutral.plainHoverBg',
+        sx={[
```

Comment:
⚠️ we shouldn't change the order of the styles.

Reply:
Here is one consequence of this: https://github.com/mui/material-ui/pull/42498/commits/d760fb5ede7d4abe6c01110717fbe3edea79e61b

Reply:
> Here is one consequence of this: d760fb5

I think that's my manual mistake for changing `theme.applyStyles('light')` to `theme.applyStyles('dark')`. Thanks for fixing it.